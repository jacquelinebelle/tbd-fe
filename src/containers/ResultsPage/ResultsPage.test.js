import React from 'react';
import {shallow} from 'enzyme';
import {ResultsPage} from './index';


describe('Results Page', () => {

  let wrapper; 
  beforeEach(() => {
    wrapper = shallow(
      <ResultsPage />
    )

  })


  it('should match the snapshot', () => {

    expect(wrapper).toMatchSnapshot()
  })
})