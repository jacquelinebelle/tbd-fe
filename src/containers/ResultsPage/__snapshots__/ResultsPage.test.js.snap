// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results Page should match snapshot 1`] = `
<main
  className="results-page"
>
  <section
    className="job-list"
  >
    <JobListing
      company="IQVIA"
      key="2489493276763186000"
      lastModified="2019-09-04T03:13:55.1458976+03:00"
      link="https://us.jooble.org/away/2489493276763186316?p=1&pos=1&cid=512&ckey=Javascript&age=134&relb=175&brelb=115&scr=7278,57184782609&bscr=4783,0615&aq=19401826"
      location="Mesa, AZ"
      salary=""
      snippet="Sample snippet 1"
      title="JavaScript Developer"
      type="Full-time"
    />
    <JobListing
      company="Vaco Technology"
      key="2476942630518357000"
      lastModified="2019-09-05T08:01:17.9188044+03:00"
      link="https://us.jooble.org/away/-2476942630518357134?p=1&pos=2&cid=512&ckey=Javascript&age=105&relb=175&brelb=115&scr=7240,01630434783&bscr=4757,725&aq=19401826"
      location="Durham, NC"
      salary=""
      snippet="Sample snippet 2"
      title="JavaScript Developer"
      type="Full-time"
    />
  </section>
</main>
`;
